examples/integration_test.py

```diff
@@ -842,6 +857,56 @@ def my_on_revoke(consumer, partitions):
     c.close()
 
 
+def verify_throttle_cb():
+    """ Verify throttle_cb is invoked """
```

Comment:
Add a reference to the tests/README for the pre-requisites / how to set this up.